test/integration/migrate/index.js

```diff
@@ -55,41 +54,16 @@ module.exports = function(knex) {
       });
 
       it('should return 0 if code matches DB', function() {
-        // Put in a couple dummy migrations. Needed
-        // since the migrations directory has a couple
-        // bad migrations that don't get loaded. This
-        // will simulate the DB and source being in sync.
-        return Promise.all([
-          knex('knex_migrations').returning('id').insert({
```

Comment:
Lots of code was removed here, is test still checking everything fine (sorry I'm a bit tired and don't understand why code was ok to be removed)?